app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] AuthorId coming from the Author model which is an alias of the User model is a foriegn key in the Comment, Like and Post models. As such the associations between the User model and the aforementioned models should be something like below 
- `has_many :posts, foreign_key: 'author_id'` 
- `has_many :comments, foreign_key: 'author_id'` 
- `has_many :likes, foreign_key: 'author_id'`